src/components/UserDashboard/ProfilePage.tsx

```diff
@@ -135,9 +142,11 @@ function ProfilePage() {
                 <button>
                     Edit Profile! <FaUserEdit />
                 </button>
-                <button>
-                    Wishlist! <FaHeart />
-                </button>
+                <a href={path}>
+                    <button >
+                        Wishlist! <FaHeart />
```

Comment:
> Also noticed that when going to the wishlist page, the sign in feature on the navbar doesn't display the profile despite being signed in?

This is because using <a> tag will request full document (CSS, JS, images) and refresh the page. 

This can be solved by using`NavLink` from React Router DOM.

Can you try replacing the current <a> tag code like this?
`<NavLink to={path}>Wishlist ! <FaHeart /></NavLink>`

Reply:
Also, would you mind adding `useAuthState();` to App.tsx like this? It was there before, but I don't see it right now.

```jsx
export default function App() {
    useAuthState();
    const [allAmiibo, setAllAmiibo] = useState(false);
    const [toggle, setToggle] = useState(false);
    const { isLoading, error } = GetAmiibo();
```